app/views/course/discussion/posts/_form.html.slim

```diff
@@ -1,4 +1,4 @@
-div.post-form style='display: none'
+div.post-form
```

Comment:
@allenwq sorry but it got removed here ... 

Just now I was reviewing on a per-commit change so ... is it intended for `display: none` to exist in the partial?